crds/console-chart/user-preference-crd.yaml

```diff
@@ -22,6 +22,28 @@ spec:
       served: true
       # One and only one version must be marked as the storage version.
       storage: true
+      schema:
+        openAPIV3Schema:
+          type: object
+          properties:
+            spec:
+              type: object
+              properties:
+                savedSearches: # Saved user searches to be displayed on main search page
+                  type: array
+                  items:
+                    type: object
+                    properties:
+                      id:
+                        type: string
+                      searchText:
+                        type: string
+                      name:
+                        type: string
+                      description:
+                        type: string
+                homepage: # User is able to set a homepage that creates a link on header icon
+                  type: string
```

Comment:
Let's remove `homepage`.  It was never implemented so we don't have to worry about migrating any dependencies.

Reply:
Removed